bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/TextLayout.java

```diff
@@ -60,12 +60,15 @@ public final class TextLayout extends Resource {
 
 	StyleItem[] allRuns;
 	StyleItem[][] runs;
-	int[] lineOffset, lineY, lineWidth;
+	int[] lineOffset, lineY;	// In Points
+	int[] lineWidth;	// In pixels
```

Comment:
Talked to Andy about it. Renaming the fields would make the diff explode. I can create a separate PR for this change otherwise it would be difficult to review.

Reply:
Perfect, do that 👍